CSharp_Net_module1_4_1_lab/CSharp_Net_module1_4_1_lab/Customer.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharp_Net_module1_4_1_lab
+{
+    class Customer
+    {
+        // 6) declare private field name
+        private string name;
+
+        // 7) declare constructor to initialize name
+        public Customer(string customerName)
+        {
+            this.name = customerName;
+        }
+
+        // 8) declare method GotNewGoods with 2 parameters:
```

Comment:
let's remove all comments.
let's give chance for code to tell us what it point for)

Reply:
I agree, those comments from template looks like not ntive)